src/content/docs/extensions/full-text-search.md

```diff
@@ -0,0 +1,201 @@
+---
+title: "Full Text Search"
+---
+
+## Usage
+
+The `FTS` (full-text search) extension adds support for matching within the content of a string property
+while returning the documents with a proximity score to the query. It is enabled by building an index
+on string properties in a table and allows searching through the strings via a keyword query.
+Currently, Kùzu supports only indexing on a node table's `STRING` properties.
+
+The FTS functionality is not available by default, so you would first need to install the `FTS`
+extension by running the following commands:
+
+```sql
+INSTALL FTS;
+LOAD EXTENSION FTS;
+```
+
+### Example dataset
+
+Let's look at an example dataset to demonstrate how the FTS extension can be used.
+First, let's create a `Book` table containing each book's information, including the title, author and abstract.
+
+```cypher
+CREATE NODE TABLE Book (ID SERIAL, abstract STRING, author STRING, title STRING, PRIMARY KEY (ID));
+CREATE (b:Book {abstract: 'An exploration of quantum mechanics.', author: 'Alice Johnson', title: 'The Quantum World'});
+CREATE (b:Book {abstract: 'A magic journey through time and space.', author: 'John Smith', title: 'Chronicles of the Universe'});
+CREATE (b:Book {abstract: 'An introduction to machine learning techniques.', author: 'Emma Brown', title: 'Learning Machines'});
+CREATE (b:Book {abstract: 'A deep dive into the history of ancient civilizations.', author: 'Michael Lee', title: 'Echoes of the Past'});
+CREATE (b:Book {abstract: 'A fantasy tale of dragons and magic.', author: 'Charlotte Harris', title: 'The Dragon\'s Call'});
+```
+
+In the following sections, we will build a full-text search index on the book table, and demonstrate how to search for books relevant to a keyword query.
+
+### Create FTS index
+
+Kuzu provides a function `CREATE_FTS_INDEX` to create the full-text search index on a table:
+
+```cypher
+CALL CREATE_FTS_INDEX('TABLE_NAME', 'INDEX_NAME', ['PROP1', 'PROP2', 'PROP3'...], OPTIONAL_PARAM1 := 'OPTIONAL_VAL1')
+```
+- `TABLE_NAME`: The name of the table to build FTS index.
+- `INDEX_NAME`: The name of the FTS index to create.
```

Comment:
Should mention here too that a table can only be of type Node.

Also mention that the same table can have multiple indexes.

Reply:
Fixed: https://github.com/kuzudb/kuzu-docs/pull/358/files